pkg/swap/swap.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
```

Comment:
absolutely not python or ruby or node, no, but we do plan to rewrite most of the shell in linuxkit in Rust/Go actually. We already have too much shell code. Also the fewer things we need to install the more we will be able to share between images once we have the `containerd` image store. Also installing `bash` in Alpine pulls in a lot of dependencies (ncurses etc), and bloats Alpine up to 12MB...